doc/dev/crimson/crimson.rst

```diff
@@ -263,11 +260,11 @@ fio
 ---
 
 ``crimson-store-nbd`` exposes configurable ``FuturizedStore`` internals as an
-NBD server for use with fio.
+NBD server for use with ``fio``.
 
-To use fio to test ``crimson-store-nbd``,
+In order to use ``fio`` to test ``crimson-store-nbd`` perform the below steps.
```

Comment:
s/``crimson-store-nbd``/``crimson-store-nbd``,/

(Add a comma between the clauses.)